main.c

```diff
@@ -228,7 +228,56 @@ void remove_log(
     struct board_tile board[SIZE][SIZE],
     int last_coordinate[2]
 ) {
-    printf("// TODO remove_log [row] [column]\n");
+    int row, col;
+    printf("To remove log(s), please input [row] [column]: ");
+    scanf("%d %d", &row, &col);
+
+    if (board[row][col].type != LOG) //Tile that is not log tile will not be changed.
+        {
+            printf("There is no log to remove at row %d and column %d.\n", row, col);
+            return;
+        }
+
+    //To check for Frogger.
+    int frogger_present = 0;
```

Comment:
nitpick, you can change this to the constant `TRUE` and `FALSE` for clearer intent

```suggestion
    int frogger_present = FALSE;
```